cypress/e2e/auddio-API-tests.cy.js

```diff
@@ -0,0 +1,151 @@
+describe('happy-path', () => {
+  describe('intial setup of app when loaded', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3000')
```

Comment:
i think we could add cy.clock here as even though it's not used with all tests it should not affect them

Reply:
done